app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/Category.kt

```diff
@@ -3,4 +3,4 @@ package com.github.HumanLearning2021.HumanLearningApp
 class Category(
     var name: String = "",
 
-)
\ No newline at end of file
+    )
```

Comment:
this whole file seems useless, would be good to delete it (we already have model/DummyCategory)

Reply:
Yes exactly what I have just suggested in the other comment with Jonas